src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/UserAcquisitionServiceImpl.java

```diff
@@ -31,7 +30,6 @@
         try (Response response = userProfileClient.findUser(bearerToken, s2sToken, id)) {
 
             ResponseEntity responseEntity = JsonFeignResponseHelper.toResponseEntity(response, GetUserProfileResponse.class);
-            Class clazz = response.status() > 300 ? ErrorResponse.class : GetUserProfileResponse.class;
```

Comment:
Sonar: "Remove this useless assignment to local variable 'clazz' " - it was never used.
https://sonarcloud.io/project/issues?id=RDPROFILESYNCAPI&open=AWxuE_h2_Xcr_PhAhRjL&resolved=false